src/JustEat.StatsD.Tests/WhenUsingPooledUdpTransport.cs

```diff
@@ -1,15 +1,26 @@
+using System;
+using System.Net;
 using System.Threading.Tasks;
+using JustEat.StatsD.EndpointLookups;
 using Xunit;
 
 namespace JustEat.StatsD
 {
-    public static class WhenUsingPooledUdpTransport
+    [Collection("ActiveUdpListeners")]
+    public class WhenUsingPooledUdpTransport 
     {
+        public WhenUsingPooledUdpTransport(UdpListeners _)
+        {
+        }
+
         [Fact]
-        public static void AMetricCanBeSentWithoutAnExceptionBeingThrown()
+        public void AMetricCanBeSentWithoutAnExceptionBeingThrown()
         {
             // Arrange
-            var endPointSource = EndpointLookups.EndpointParser.MakeEndPointSource("127.0.0.1", 8125, null);
+            var endPointSource = EndpointParser.MakeEndPointSource(
+                UdpListeners.EndpointA.Address.ToString(),
```

Comment:
This makes me wonder whether we should have an overload that accepts an `IPEndPoint`.

Reply:
What do you think @AnthonySteele ?

Reply:
So the fix for the tests on mac and linux was purely in the test setup?

Reply:
#92